### DISCUSSION TITLE
Wallet BUG

### DISCUSSION THREAD

**Discussion by aceyvr** [Chia-Network/chia-blockchain#8914] [Support]
Hello!

My Wallet address xch1pa02l3tw6l43xr2dtmvccym0hmnnxxtylfml6zv75qw7glwqw3pq064eff set to receive Block Rewards and i have won 4 blocks so far and I only seeing zero Balance in my fully synced wallet. Please help. Thanks

**Comment by shaneo257** [comment]
i've had a look at xch scan and that address isn't valid
https://xchscan.com/address/xch1pa02l3tw6l43xr2dtmvccym0hmnnxxtylfml6zv75qw7glwqw3pq064eff

**Comment by Mortezan20** [comment]
Hi, 3 days ago, I sent some xch currency from Quinx Exchange to a valid chia wallet, but it still shows zero after 3 days. Please help
0x24cc8260e1545a0de9dd3246950769ee66db1f703562df16268e8fb4c53642a5
